arithmetization/src/main/java/net/consensys/linea/zktracer/runtime/callstack/CallFrame.java

```diff
@@ -203,6 +203,7 @@ public void rememberGasNextBeforePausing(Hub hub) {
     this.parentId = parentId;
     this.callDataRange = callDataRange;
     this.returnAtRange = returnAtRange;
+    this.outputDataRange = new MemoryRange(contextNumber);
```

Comment:
The `outputDataRange`, if not overwritten nontrivially at `traceContextExit`, should at least remember its own context number, as that will always be correct and provides its parent frame at least with the correct `returnDataContextNumber` by means of `returnDataRange.contextNumber()`.